src/machine/asm/execute_aarch64.S

```diff
@@ -52,14 +53,16 @@
 #endif
 
 #define PREPCALL \
-  stp x0, x9, [sp, -48]! SEP \
```

Comment:
I think `x0` here is only previously used as a placeholder to make the stack aligned, can you test if it works when we only save x9 - x14, which is exactly 6 registers in 48 bytes?

Reply:
x0 stores the address of the Machine, I don't think it is used as a placeholder.

```
#define MACHINE x0
```